src/components/dashboard/ReceiveSummary.js

```diff
@@ -72,4 +73,12 @@ ReceiveAmount.shouldNavigateToComponent = props => {
   return screenState.amount
 }
 
-export default withStyles()(ReceiveAmount)
+const getStylesFromProps = ({ theme }) => ({
```

Comment:
Why not put all this styles into BackButton? Seems like all the BackButtons should look the same

Reply:
done! worked well so far, thanks!